### PR TITLE
refactor(ADT,Tests):Get the models after creating them to account for a delay from the service

### PR DESCRIPTION
**Pull request by barustum**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ComponentTests.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ public async Task Component_Lifecycle()
                 // create wifi model
                 string roomWithWifiModel = TestAssetsHelper.GetRoomWithWifiModelPayload(roomWithWifiModelId, wifiModelId, wifiComponentName);
 
-                await client.CreateModelsAsync(new List<string> { roomWithWifiModel, wifiModel }).ConfigureAwait(false);
+                await createAndListModelsAsync(client, new List<string> { roomWithWifiModel, wifiModel }).ConfigureAwait(false);
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ComponentTests.cs:42]: This is javaCasing bro :p

File: sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs [hunk 2]
```diff
@@ -92,5 +93,17 @@ protected string GetRandom()
         {
             return Recording.GenerateId();
         }
+
+        protected async Task createAndListModelsAsync(DigitalTwinsClient client, List<string> lists)
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs:97]: worth a comment we do this to work around a bug or issue of latency

**Review by drwill-ms — APPROVED** [review]

File: sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ComponentTests.cs (hunk 1 above)
Comment (barustum) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ComponentTests.cs:42]: Tim and Azad are getting into me with this whole Java thing..sigh

File: sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs (hunk 2 above)
Comment (barustum) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs:97]: good idea

**Pull request merged**